### PR TITLE
fix incorrect github link

### PR DESCRIPTION
**Pull request by Tompage1994**
### What does this PR do?
Small fix for the readme links which were pointing to the wrong github URL (controller_configuration.git)

e.g. note that the status icons on the readme are currently broken

### How should this be tested?
N/A

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,16 +26,16 @@ Click the `Content` button to see the list of content included in this collectio
 
 ## Installing this collection
 
-You can install the redhat_cop controller_configuration collection with the Ansible Galaxy CLI:
+You can install the redhat_cop tower_configuration collection with the Ansible Galaxy CLI:
```
Comment (sean-m-sullivan) [inline README.md]: The links should change, the text should not, we should keep calling it controller configuraiton, but the web links should be tower configuration.

**Review by sean-m-sullivan — CHANGES_REQUESTED** [review]
Need to change back in several places, otherwise agree.

File: README.md (hunk 1 above)
Comment (Tompage1994) [reply README.md]: My bad. Just did a dumb find and replace without thinking

**Review by sean-m-sullivan — APPROVED** [review]
LGTM

**Pull request merged**